src/cli/admin/admin-federation-export.ts

```diff
@@ -7,86 +7,90 @@ import {
   exportAdminFederationProviderToFile,
 } from '../../ops/cloud/AdminFederationOps';
 import { printMessage, verboseMessage } from '../../utils/Console';
-import { FrodoCommand } from '../FrodoCommand';
 
-const program = new FrodoCommand('frodo admin federation export', ['realm']);
-
-program
-  .description('Export admin federation providers.')
-  .addOption(
-    new Option(
-      '-i, --idp-id <idp-id>',
-      'Id/name of a provider. If specified, -a and -A are ignored.'
+export const addFederationExport = (command: Command) => {
+  command
+    .command('export')
```

Comment:
I know there's some magic of adding default options and omitting some and other stuff which we'd have to extract into a helper function here.

The original uses implementation inheritance, where it might be easier to think of the standard options as things we can apply with functions.